src/components/BaseModalSelectItem.vue

```diff
@@ -15,7 +15,7 @@ defineProps<{
     ]"
   >
     <div class="relative inset-y-0 flex items-center">
-      <div :class="['w-full', { 'pr-[44px]': selected }]">
+      <div :class="['w-full text-left', { 'pr-[44px]': selected }]">
```

Comment:
Any reason for text-left here? it is left without it also no? 🤔

Reply:
Nope, this was a bug I discovered and added to this PR. Will hotfix in on prod now instead.
<img width="589" alt="image" src="https://user-images.githubusercontent.com/51686767/233904339-8ec48199-f03e-4287-a93d-883296ffe840.png">